app/wp-content/themes/kstg/library/scss/style.scss

```diff
@@ -4,6 +4,7 @@
 @import "components/base";
 @import "components/typography";
```

Comment:
Oh, I also noticed that 'base' is being imported within typography. May want to remove that so that the 'base' scss isn't being imported twice.